README.md

```diff
@@ -854,6 +854,43 @@
         .call(tron.led);
     ```
 
+  - Leave a blank line after blocks and before the next statement
+
+    ```javascript
+    // bad
+    if (true) {
+      var a = 2;
+    }
+    var b = 3;
+
+    // good
+    if (true) {
+      var a = 2;
```

Comment:
1. Not a good idea to declare a var inside if block under `good` sign
2. Actually, no difference from `bad`